exonum-light-client/README.md

```diff
@@ -9,14 +9,25 @@ Also, Exonum light client provides access to [common utils][ejb-common]
 toolkit which contains some helpful functions for _hashing_,
 _cryptography_, _serialization_ etc. 
 
+## Capabilities
+By using the client you are able to perform next operations:
+- Submit transactions to the node
+- Receive transaction information 
+- Receive blockchain blocks information 
+- Receive node system information 
```

Comment:
What is that and how is it different from the node status information below?

Reply:
os, exonum versions vs health check info